ast/marshal/json.go

```diff
@@ -0,0 +1,33 @@
+package marshal
+
+// JSONOptions defines the options for JSON operations,
+// currently only marshaling can be configured
+type JSONOptions struct {
+	MarshalOptions JSONMarshalOptions
```

Comment:
Since the package is named `marshal`, I feel like this could be flattened, like
```go
type JSONOptions struct {
	// IncludeLocation toggles the marshaling of location information
	IncludeLocation NodeToggle
	// IncludeLocationText additionally/optionally includes the text of the location
	IncludeLocationText bool
}
```

Reply:
I guess I had in my mind that we _might_ want to also have marshalling options by included in here. The idea being that that we could add an UnmarshalOptions field to JSONOptions without breaking it users of the original type.


Perhaps a better name for the package is just json?